libraries/networking/src/UserActivityLogger.h

```diff
@@ -35,7 +35,11 @@ public slots:
     bool isEnabled() { return !_disabled.get(); }
     bool isDisabledSettingSet() const { return _disabled.isSet(); }
 
+    bool isCrashEnabled() { return !_crashDisabled.get(); }
+    bool isCrashDisabledSettingSet() const { return _crashDisabled.isSet(); }
+
     void disable(bool disable);
+    void crashDisable(bool disable);
```

Comment:
perhaps rename `crashDisable` to `disableCrashReporting` . 

- `crashDisable` sounds like you're crashing a disable which doesn't make sense